form/events.rst

```diff
@@ -11,7 +11,7 @@ Using form events, you may modify information or fields at different steps
 of the workflow: from the population of the form to the submission of the
 data from the request.
 
-Registering an event listener is very easy using the Form component.
+Registering an event listener is straightforward using the Form component.
```

Comment:
I'm not sure this is correct. I don't find it easy nor straight forward and I always have to peek into the docs for this. I think this whole sentence needs a rewrite 🤔

Reply:
Hmm 🤔

"Registering an event listener is also possible using the Form component" ?

Reply:
This one is difficult...

"By utilizing the Symfony event system, event listeners can be registered to ... "

Something like that perhaps?

Reply:
ping @xabbuh @javiereguiluz @weaverryan

Reply:
I like the suggestion from https://github.com/symfony/symfony-docs/pull/11044#discussion_r259612932

Reply:
Done, are you fine with it? @linaori